sharepoint-adfs/azuredeploy.json

```diff
@@ -0,0 +1,855 @@
+      }
+    },
+    "spAppPoolUserName": {
+      "type": "string",
+      "defaultValue": "spapppool",
+      "minLength": 1,
+      "metadata": {
+        "description": "Name of the service account that will run SharePoint web applications"
+      }
+    },
+    "spAppPoolPassword": {
+      "type": "securestring",
+      "minLength": 8,
+      "metadata": {
+        "description": "Input must meet password complexity requirements as documented for property 'adminPassword' in https://docs.microsoft.com/en-us/rest/api/compute/virtualmachines/virtualmachines-create-or-update"
+      }
+    },
+    "spPassphrase": {
+      "type": "securestring",
+      "minLength": 1,
+      "metadata": {
+        "description": "Passphrase of the SharePoint farm"
+      }
+    },
+    "vmDCName": {
+      "type": "string",
+      "defaultValue": "DC",
+      "minLength": 2,
+      "metadata": {
+        "description": "Internal name of DC VM"
+      }
+    },
+    "vmDCSize": {
+      "type": "string",
+      "defaultValue": "Standard_F4",
+      "metadata": {
+        "description": "Size of the DC VM"
+      }
+    },
+    "vmDCPublicIPDnsName": {
+      "type": "string",
+      "minLength": 2,
+      "metadata": {
+        "description": "'Input' used to create public DNS name 'Input.region.cloudapp.azure.com' of DC VM"
+      }
+    },
+    "vmSQLName": {
+      "type": "string",
+      "defaultValue": "SQL",
+      "minLength": 2,
+      "metadata": {
+        "description": "Internal name of SQL VM"
+      }
+    },
+    "vmSQLSize": {
+      "type": "string",
+      "defaultValue": "Standard_A4_v2",
+      "metadata": {
+        "description": "Size of the SQL VM"
+      }
+    },
+    "vmSQLPublicIPDnsName": {
+      "type": "string",
+      "minLength": 2,
+      "metadata": {
+        "description": "'Input' used to create public DNS name 'Input.region.cloudapp.azure.com' of SQL VM"
+      }
+    },
+    "vmSPName": {
+      "type": "string",
+      "defaultValue": "SP",
+      "minLength": 2,
+      "metadata": {
+        "description": "Internal name of SP VM"
+      }
+    },
+    "vmSPSize": {
+      "type": "string",
+      "defaultValue": "Standard_A8_v2",
+      "metadata": {
+        "description": "Size of the SQL VM"
+      }
+    },
+    "vmSPPublicIPDnsName": {
+      "type": "string",
+      "minLength": 2,
+      "metadata": {
+        "description": "'Input' used to create public DNS name 'Input.region.cloudapp.azure.com' of SP VM"
+      }
+    },
+    "vmsTimeZone": {
+      "type": "string",
+      "defaultValue": "Central European Standard Time",
+      "minLength": 2,
+      "metadata": {
+        "description": "Time zone of the VMs. Get the list with '[System.TimeZoneInfo]::GetSystemTimeZones().Id'. Note that 'UTC' works but 'UTC+xx' does NOT work."
+      }
+    },
+    "dscDCTemplateURL": {
```

Comment:
I think I set DSC URL as mentioned in https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#rawgithubusercontentcom-links
Is the URL wrong?